app/assets/stylesheets/application/variables.css

```diff
@@ -1,47 +0,0 @@
-:root {
-  --color-text: #272d3b;
-  --color-text-muted: #a8aaad;
-  --color-text-label: #a8aaad;
-  --color-text-muted-dark-background: #686d72;
-
-  --color-nav: #ffffff;
-  --color-warning: #ff5167;
-  --color-error: #ff5167;
-  --color-background: #f4f8fa;
-  --color-muted-background: #edeeee;
-  --color-background-dark: #191d24;
-  --color-background-dark-alternate: #17191d;
-  --color-tint: #6750a4;
-  --color-warn: #f39c12;
-  --color-border: rgba(0 0 0 /0.1);
-  --color-footer: #1d2024;
-  --color-shadow: rgba(--color-text, 0.3);
-
-  --size-text: 14px;
-  --size-safe-inset: 0px;
-  --size-safe-inset: env(safe-area-inset-left);
-  --size-safe-padding: 8px;
-  --size-safe-padding: max(env(safe-area-inset-right), 8px);
-  --size-safe-bottom: 0px;
-  --size-safe-bottom: max(env(safe-area-inset-bottom), 0px);
-  --size-tab-bar: calc(49px + var(--size-safe-bottom));
-  --size-sidebar: 250px;
-
-  --font-text: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
```

Comment:
This variable is still used in the `reset.scss`. Something with the font selection is broken, because much of my UI is rendered in a serif font and I assume that's not on purpose.

![image](https://github.com/user-attachments/assets/87788376-80a1-4b69-a007-b1db3f810542)

Reply:
The serif font is actually on purpose to show us instances of things that don't use the `<Text>` component (mostly because there's no design yet)